### DISCUSSION TITLE
Unfortunately, we weren't able to approve your educational discount request for @Umar04R.   What happened? Suggested proof to expedite your approval:  Attendees of this school must verify and use their academic email address. When you have verified your school's academic email address on your account, ensure that you select the school when prompted on the application page.

### DISCUSSION THREAD

**Discussion by Umar04R** [community/community#137151] [GitHub Education]
### Hi everyone,

Unfortunately, we weren't able to approve your educational discount request for [@Umar04R](https://github.com/Umar04R).

What happened?
Suggested proof to expedite your approval:

Attendees of this school must verify and use their academic email address. When you have verified your school's academic email address on your account, ensure that you select the school when prompted on the application page.

**Comment by davevad93** [comment]
Hi @Umar04R ,

That means you first have to verify your academic email address in your GitHub account, to do that go to the [email settings](https://github.com/settings/emails), add your academic email if you haven't already and verify it, once verified go back to the [application page](https://education.github.com/discount_requests/application) and select the school when prompted. 

If you continue to experience problems or rejections with the application afterwards, be sure to check the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352), you'll find a lot of useful info there.

**Reply by Umar04R** [reply]
thanks

**Reply by davevad93** [reply]
You're welcome @Umar04R
